opengever/maintenance/scripts/set_preserved_as_paper_to_false.py

```diff
@@ -0,0 +1,58 @@
+"""
+Script to set `preserved_as_paper` to False for all `IDocumentMetadata`.
+
+    bin/instance run set_preserved_as_paper_to_false.py
+
+"""
+from opengever.document.behaviors.metadata import IDocumentMetadata
+from opengever.maintenance.debughelpers import setup_app
+from opengever.maintenance.debughelpers import setup_option_parser
+from opengever.maintenance.debughelpers import setup_plone
+from plone import api
+import logging
+import transaction
+
+
+logger = logging.getLogger("set_preserved_as_paper_to_false")
+logging.getLogger().setLevel(logging.INFO)
+for handler in logging.getLogger().handlers:
+    handler.setLevel(logging.INFO)
+
+
+def set_preserved_as_paper_to_false():
+    catalog = api.portal.get_tool("portal_catalog")
+    logger.info("Setting preserved_as_paper to False")
+    query = {
+        "object_provides": IDocumentMetadata.__identifier__,
+    }
+    brains = catalog.unrestrictedSearchResults(query)
+    total = len(brains)
+
+    for i, brain in enumerate(brains):
+        obj = brain.getObject()
+        meta = IDocumentMetadata(obj)
+        meta.preserved_as_paper = False
+
```

Comment:
I'm not sure, but is there no catalog or solr index?

Reply:
I was also surprised, but no. There is no index, no catalog metadata and no Solr field.